source/guides/puppetlabs_package_repositories.markdown

```diff
@@ -1,5 +1,5 @@
 ---
-title: "Using the Puppet Labs Package Repositories"
+title: "Using Puppet Collections"
```

Comment:
These are band-aid edits to the unversioned guide while we continue work on the versioned editions.